language/vm/vm-runtime/src/txn_executor.rs

```diff
@@ -91,32 +77,36 @@ impl<'txn> TransactionExecutor<'txn> {
         )
     }
 
+    #[allow(non_snake_case)]
+    pub fn publish_module_FOR_GENESIS_ONLY(
```

Comment:
this is kind of horrible but I am going to rework the genesis soon to use the runtime in a more proper way